### PR TITLE
New Package: python3-libclang-3.9.0

### PR DESCRIPTION
**Pull request by mfulz**

### CODE REVIEW

File: srcpkgs/python3-libclang/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+checksum=ab1885be148303c71d35105cb55f4c370126e690458143474e9a44b4ebe05f3c
+wrksrc="libclang-py3-${version}"
+
+#python3-keyutils_package() {
+	#pycompile_module="keyutils"
+	#short_desc="${short_desc/Python2/Python3}"
+	#pkg_install() {
+		#vmove usr/lib/python3*
+	#}
+#}
```
Comment (SirDoctors) [inline srcpkgs/python3-libclang/template]: Why is this here if its all commented out?

File: srcpkgs/python3-libclang/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+revision=1
+build_style=python3-module
+pycompile_module="libclang"
+python_version=3
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+short_desc="Python3 bindings for clang"
+maintainer="Matthias Fulz <mfulz@olznet.de>"
+homepage="https://bitbucket.org/Anteru/python3-libclang"
+license="custom:University of Illinois/NCSA Open Source License"
```
Comment (SirDoctors) [inline srcpkgs/python3-libclang/template]: Please use ``vlicense`` to install this license with the package. (information on vlicense can be found [here](https://github.com/voidlinux/void-packages/blob/master/Manual.md#global-functions).

Comment (mfulz) [inline srcpkgs/python3-libclang/template]: It is not inside the sources. I'll check where I can optain it.

File: srcpkgs/python3-libclang/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+distfiles="https://pypi.python.org/packages/aa/73/d0cc2b5e896bf6ea152b2324d00963ec6b0b116bb5ed672018c0bacd97c0/libclang-py3-${version}.tar.gz"
```
Comment (SirDoctors) [inline srcpkgs/python3-libclang/template]: Please use ``${PYPI_SITE}/l/libclang-py3/libclang-py3-${version}.tar.gz`` instead.

File: srcpkgs/python3-libclang/template (hunk 2 above)
Comment (SirDoctors) [inline srcpkgs/python3-libclang/template]: Alright then, when you find it just put the license in ``files/LICENSE``.

Comment (mfulz) [inline srcpkgs/python3-libclang/template]: I just found this: https://opensource.org/licenses/NCSA
and this seems to be the author: https://github.com/Anteru

Shall I just add the author to the license and put it under files/LICENSE?

Comment (SirDoctors) [inline srcpkgs/python3-libclang/template]: Fill in what you can of the license and then add it.
```
Copyright (c) <Year> <Owner Organization Name> 
All rights reserved.

Developed by: 		<Name of Development Group>
                      	<Name of Institution>
                        <URL for Development Group/Institution>
Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal with the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:

Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimers.
Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimers in the documentation and/or other materials provided with the distribution.
Neither the names of <Name of Development Group, Name of Institution>, nor the names of its contributors may be used to endorse or promote products derived from this Software without specific prior written permission.
THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE CONTRIBUTORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS WITH THE SOFTWARE.```

Comment (mfulz) [inline srcpkgs/python3-libclang/template]: Done

File: srcpkgs/python3-libclang/files/LICENSE [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Copyright (c) 2017
+All rights reserved.
+
+Developed by: 		Matthäus G. Chajdas
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal with the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimers.
+Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimers in the documentation and/or other materials provided with the distribution.
+Neither the names of <Name of Development Group, Name of Institution>, nor the names of its contributors may be used to endorse or promote products derived from this Software without specific prior written permission.
```
Comment (SirDoctors) [inline srcpkgs/python3-libclang/files/LICENSE]: Replace the text in <> here.

File: srcpkgs/python3-libclang/files/LICENSE [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Copyright (c) 2017
```
Comment (SirDoctors) [inline srcpkgs/python3-libclang/files/LICENSE]: Add the author's name right after the year.

File: srcpkgs/python3-libclang/template [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'python-libclang'
+pkgname=python3-libclang
+version=3.9.0
+revision=1
```
Comment (asergi) [inline srcpkgs/python3-libclang/template]: `noarch=yes`

File: srcpkgs/python3-libclang/template [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+build_style=python3-module
+pycompile_module="libclang"
```
Comment (asergi) [inline srcpkgs/python3-libclang/template]: `pycompile_module="clang"`

File: srcpkgs/python3-libclang/template [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+python_version=3
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
```
Comment (asergi) [inline srcpkgs/python3-libclang/template]: Remove `makedepends` and add `depends="clang"`

File: srcpkgs/python3-libclang/template [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+short_desc="Python3 bindings for clang"
+maintainer="Matthias Fulz <mfulz@olznet.de>"
+homepage="https://bitbucket.org/Anteru/python3-libclang"
+license="custom:University of Illinois/NCSA Open Source License"
```
Comment (asergi) [inline srcpkgs/python3-libclang/template]: Remove `custom:` from `license`.

**Comment by asergi** [comment]
Stalled for a year. Feel free to resubmit it.
